src/reversible_tokenize.jl

```diff
@@ -0,0 +1,75 @@
+# A simple reversible tokenizer
+"""
+A simple reversible tokenizer
+
+```
+tokenized = rev_tokenizer(instring)
+de_tokenized = rev_detokenizer(token)
+```
+
+The rev_tokenize tokenizer splits into token based on space and punctuations and in 
+addition it leaves some MERGESYMBOLS for the tokens to be re-arranged when needed 
+using the rev_detokenize.
+It uses a character based approach for splitting and re-merging.
+
+Parameters:
+
+- instring		= Input string to be tokenized 
+- token			= Collection to tokens i.e String Array
+
+"""
+MERGESYMBOL = '⇆'
```

Comment:
I still think the symbol should be from the private use area.
(i.e. characters that on most font's will not even display and that will not display consistantly between fonts.

Also this line needs to be declared `const`

Reply:
https://en.wikipedia.org/wiki/Private_Use_Areas